free-courses-en.md

```diff
@@ -210,6 +213,7 @@
 
 ### Ruby
 
+* [Learn Ruby the Hard Way](http://learnrubythehardway.org/book/)
```

Comment:
Is already there. We have as far as i know all books of the "Learn xy the hard way"-franchise